build/docker/pbs-cluster/supervisord.conf

```diff
@@ -0,0 +1,14 @@
+[supervisord]
+nodaemon=true
+
+[program:sshd]
+command=/usr/sbin/sshd -D
+priority=10
+autostart=true
+autorestart=true
+
+[program:init-script]
+command=/usr/local/bin/init_sequence.sh
```

Comment:
Instead of starting multiple services within the init_sequence.sh would it be possible to start them directly here?
The goal of supervisord is managing services including dependencies.